packages/okta-react/src/SecureRoute.js

```diff
@@ -14,6 +14,32 @@ import React, { Component } from 'react';
 import { Route } from 'react-router';
 import withAuth from './withAuth';
 
+
+class RenderWrapper extends Component {
```

Comment:
Any reason you decided to include this class inside of this file? Seems like we could pull this out.

Reply:
Because it is only used within this file, and it is strongly related to "SecureRoute". But feel free to pull it out into its own file.

Reply:
Agreed, it's so coupled it's easier to just see it in the same file